test/expression/test_aggregation.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+
+from src.expression.abstract_expression import ExpressionType
+from src.expression.comparison_expression import ComparisonExpression
+from src.expression.aggregation_expression import AggregationExpression
+from src.expression.constant_value_expression import ConstantValueExpression
+from src.expression.tuple_value_expression import TupleValueExpression
+
+
+class LogicalExpressionsTest(unittest.TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def test_aggregation_sum(self):
+        columnName = TupleValueExpression(0)
+        aggr_expr = AggregationExpression(
+            ExpressionType.AGGREGATION_SUM,
+            None,
+            columnName
+        )
+        tuple1 = [[1, 2, 3], [2, 3, 4], [3, 4, 5]]
```

Comment:
We are moving from this type of tuple to actual video frames. So we should stop merging more code with this kind of test cases.